tooling/profiler/src/cli/mod.rs

```diff
@@ -19,15 +20,17 @@ struct ProfilerCli {
 #[non_exhaustive]
 #[derive(Subcommand, Clone, Debug)]
 enum ProfilerCommand {
-    GatesFlamegraph(gates_flamegraph_cmd::GatesFlamegraphCommand),
-    OpcodesFlamegraph(opcodes_flamegraph_cmd::OpcodesFlamegraphCommand),
+    Gates(gates_flamegraph_cmd::GatesFlamegraphCommand),
+    Opcodes(opcodes_flamegraph_cmd::OpcodesFlamegraphCommand),
+    ExecutionOpcodes(execution_flamegraph_cmd::ExecutionFlamegraphCommand),
```

Comment:
You can keep the names backwards compatible using [aliases](https://github.com/clap-rs/clap_derive/blob/acea444f962a0d7c8126273739d4df8bdb152624/examples/subcommand_aliases.rs#L10-L13)

Reply:
I felt it was fine to rename them fully for now anyway as the flamegraph suffix felt unnecessary (e.g. `opcodes-flamegraph` is more verbose than `opcodes`) and all the commands just return flamegraphs anyway. If the `noir-profiler` ever does anything other than generate flamegraphs we can rename them then I feel. Noted for the future though.